flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessage.java

```diff
@@ -666,10 +663,7 @@ ByteBuf write(ByteBufAllocator allocator) throws IOException {
 			ByteBuf result = null;
 
 			try {
-				result = allocateBuffer(allocator, ID, 16 + 16 + 4 + 16);
-
-				partitionId.getPartitionId().writeTo(result);
-				partitionId.getProducerId().writeTo(result);
+				result = allocateBuffer(allocator, ID, 4 + 16);
```

Comment:
nit: replace numbers with constants `Integer.BYTES` and `Long.BYTES`?

Reply:
Yes, it seems more readable for this change, but not quite relevant with this PR purpose.
After double checking the codes, there are many other places to take the current way in this class. So I might consider this change as a separate issue for unification future.